server/routes/api/user.js

```diff
@@ -0,0 +1,30 @@
+// require('dotenv').config();
+const express = require('express');
+const db = require('../../db');
+
+const router = express.Router();
+
+router.post('/favorite/:id', async (req, res) => {
+    const { id } = req.params;
+
+    const userId = req.user;
```

Comment:
`Not` req.body.user? Suspicious behaviour 🤔

Reply:
Look at what @SteveVaknin  `Wrote below on session`... maybe that would help.

Reply:
I saw what he wrote, that's the way to do it.
look at 'auth' file line 33. thats what steve wrote